tracer/test/Datadog.Trace.Tests/Agent/DiscoveryServiceTests.cs

```diff
@@ -303,7 +303,7 @@ public async Task HandlesFailuresInApiWithBackoff()
 
         await ds.DisposeAsync();
         // add some leeway in case of slowness
-        factory.RequestsSent.Count.Should().BeInRange(5, 6, "Should make at most 6 retries in 13s");
+        factory.RequestsSent.Count.Should().BeInRange(3, 6, "Should make at most 6 retries in 13s");
```

Comment:
I'm not sure what the comment should be, but I think it also needs to be updated